packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx

```diff
@@ -24,4 +32,42 @@ export class ViewerNavigationToolsProvider extends StandardNavigationToolsProvid
       },
     });
   }
+
+  public override provideToolbarButtonItems(
+    stageId: string,
+    stageUsage: string,
+    toolbarUsage: ToolbarUsage,
+    toolbarOrientation: ToolbarOrientation
+  ): CommonToolbarItem[] {
+    const items: CommonToolbarItem[] = super.provideToolbarButtonItems(
+      stageId,
+      stageUsage,
+      toolbarUsage,
+      toolbarOrientation
+    );
+
+    if (
```

Comment:
do you not get any of the other items now?

Reply:
You mean the toolbar button items? They're still there. I just had to hide the old walk tool and show the new one.
![image](https://user-images.githubusercontent.com/51834103/177852701-47aec501-2081-4613-89dd-a53e5947b926.png)